section_sd_of_natural_languages/concept_idtfs.scs

```diff
@@ -0,0 +1,4 @@
+sc_node_not_relation -> language;;
+language => nrel_main_idtf:
+[язык](* <-lang_ru;; *);;
+
```

Comment:
Not very good idea to separate idtfs in another file, possibly may cause problems with idtf search and editing in future. But for this project we may select this way of identifiers storage

Reply:
I thought this is temporary solution. Will be fixed with refactoring of the architcture.